docs/spfx/update-latest-packages.md

```diff
@@ -0,0 +1,116 @@
+# Update SharePoint Framework packages 
+
+>**Note:** The SharePoint Framework is currently in preview and is subject to change. SharePoint Framework client-side web parts are not currently supported for use in production environments.
+
+SharePoint client-side development tools use the [npm](https://www.npmjs.com/) package manager to manage dependencies and other required JavaScript helpers. npm is typically included as part of Node.js setup.
+
+When you create a new client-side solution, the yeoman generator for SharePoint fetches the latest SharePoint Framework packages required for your client-side project. As you build your project, your existing packages could be outdated as there could be new versions of one or more packages available. Based on the [release notes](https://aka.ms/spfx-release-notes) for a particular release or the latest package, you may decide to update your SharePoint Framework packages used in your project. SharePoint Framework packages include both the npm packages you have installed in your project, for example: [@microsoft/sp-core-library](https://www.npmjs.com/package/@microsoft/sp-core-library), and npm packages installed globally, for example: [@microsoft/generator-sharepoint](https://www.npmjs.com/package/@microsoft/generator-sharepoint). 
+
+While it may not be required, it is recommended you update the SharePoint Framework packages every so often so that you can get the latest changes and fixes that have been released. 
+
+## Find outdated packages in your project
+To find the outdated packages in your client-side project, including SharePoint Framework and other packages your project depends on, run the following command in a console in the same directory as your project. 
+
+```
+npm outdated
+```
+
+The command will list the following information about the packages your project depends on. This information is looked up from the `package.json` file located in the root of your project directory and npm registry.
+
+* Current version installed in your project
+* Version requested by your project (available in `package.json`)
+* Latest version available
+
+![NPM outdated packages](../../images/npm-outdated-packages-list.png)
+
+To identify the SharePoint Framework packages, look for the package names that start with the following npm scope and prefix:
+
+```
+@microsoft/sp-
```

Comment:
They will also need to update the react & office-ui-fabric-react versions. We should probably also mention these versions in the release notes.